### DISCUSSION TITLE
Welcome to next-auth-mongodb Discussions!

### DISCUSSION THREAD

**Discussion by Sylvarithos** [Sylvarithos/next-auth-mongodb#4] [Q&A] [answered]
Hello. I have a problem at deploying project.
I used Next.js+ MongoDB Atlas.
then when I send request, API returns 504 error.
Some one helps me. Thanks.

**Comment by GodricGryffindorK** [comment, marked as answer]
HI @devtopstar0201 ,
Did you connected with Vercel and MongoDB?

**Reply by Sylvarithos** [reply]
I didn't.
thanks. it works well now.
